pkg/distro/fedora/images.go

```diff
@@ -353,9 +353,9 @@ func imageInstallerImage(workload workload.Workload,
 		img.UnattendedKickstart = instCust.Unattended
 	}
 
-	// Enable anaconda-webui for Fedora > 38
+	// Enable anaconda-webui for Fedora >= 39
```

Comment:
this comment still should not be here at all, since we are removing the lines that do anything about webui here.

Reply:
I believe the enabling of the `AdditionalAnacondaModules` is still related to the webui but let me verify if that is still necessary.